src/reactive-membrane.ts

```diff
@@ -68,6 +68,24 @@ const defaultValueMutated: ReactiveMembraneMutationCallback = (obj: any, key: Pr
 };
 const defaultValueDistortion: ReactiveMembraneDistortionCallback = (value: any) => value;
 
+export function wrapDescriptor(membrane: ReactiveMembrane, descriptor: PropertyDescriptor, getValue: (membrane: ReactiveMembrane, originalValue: any) => any): PropertyDescriptor {
+    const { set, get } = descriptor;
+    if ('value' in descriptor) {
+        descriptor.value = getValue(membrane, descriptor.value);
+    }
+    if (!isUndefined(get)) {
+        descriptor.get = function() {
```

Comment:
TODO: this has the problem of the identity of the getter, asking for the descriptor twice for a configurable descriptor will give you different get functions, not a big deal, but solvable with a weakmap.

Reply:
We will track this in another issue after discussing it with @pmdartus .